invenio_app_ils/api.py

```diff
@@ -53,4 +54,10 @@ def item_exists(item_pid):
 
 def patron_exists(patron_pid):
     """Return True if the Patron exists given a PID."""
-    return True
+    try:
+        patron = User.query.filter_by(id=patron_pid).first()
+        if patron:
+            return True
+    except PersistentIdentifierError as ex:
```

Comment:
This exception will be never thrown. You might have an SQLAlchemy exception, but with `first()` it will return None if not found.

Maybe something similar to:
```
def patron_exists(patron_pid):
    return User.query.filter_by(id=patron_pid).first() != None
```

Reply:
This resolves my comment 👍

Reply:
👍